client/state/countries/reducer.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { SMS, DOMAIN, PAYMENT } from './constants';
+import {
+	COUNTRIES_RECEIVE,
+	COUNTRIES_REQUEST,
+	COUNTRIES_REQUEST_FAILURE,
+	COUNTRIES_REQUEST_SUCCESS
+} from 'state/action-types';
+import { countriesSchema } from './schema';
+import { createReducer } from 'state/utils';
+
+export const items = createReducer( { [ SMS ]: [], [ DOMAIN ]: [], [ PAYMENT ]: [] }, {
```

Comment:
Initial state looks a bit complicated. Perhaps consider breaking this into smaller reducers.
eg
```javascript
combineReducers( {
    SMS: smsReducer,
    DOMAIN: domainReducer,
    PAYMENT: paymentReducer
//...
```